tests/e2e/util.go

```diff
@@ -7029,3 +7025,268 @@ func removeStoragePolicyQuota(ctx context.Context, restClientConfig *rest.Config
+		var bytes int64 = diskSize
+		// Convert to Gi
+		//kibibytes := float64(bytes) / 1024
+		diskSize = int64(bytes) / 1024
+		fmt.Printf("diskSize:  %dGi\n", diskSize)
+	}
+
+	quota := quotaBefore - diskSize
+
+	if string(suffix1) == "Gi" {
+		var bytes int64 = quota
+		// Convert to Gi
+		//kibibytes := float64(bytes) / 1024
+		quota = int64(bytes) / 1024
+		fmt.Printf("value:  %dGi\n", quota)
+	}
+
+	fmt.Printf("diskSize %v ", diskSize)
+	fmt.Printf("cleanup quota : %v, quotaAfter: %v ", quota, quotaAfter)
+	if quota == quotaAfter {
+		validTotalQuota = true
+		ginkgo.By(fmt.Sprintf("quotaBefore - diskSize: %v, quotaAfter : %v", quota, quotaAfter))
+		ginkgo.By(fmt.Sprintf("validTotalQuota on storagePolicy:  %v", validTotalQuota))
+
+	}
+
+	return validTotalQuota
+}
+
+// validate_reservedQuota_afterCleanUp  after the volume goes to bound state or
+// after teast clean up , expected reserved quota should be "0"
+func validate_reservedQuota_afterCleanUp(ctx context.Context, total_reservedQuota *resource.Quantity,
+	policy_reservedQuota *resource.Quantity, storagepolicyUsage_reserved_Quota *resource.Quantity) bool {
+	ginkgo.By(fmt.Sprintf("reservedQuota on total storageQuota CR: %v"+
+		"storagePolicyQuota CR: %v, storagePolicyUsage CR: %v ",
+		total_reservedQuota.String(), policy_reservedQuota.String(), storagepolicyUsage_reserved_Quota.String()))
+	return total_reservedQuota.String() == "0" &&
+		policy_reservedQuota.String() == "0" &&
+		storagepolicyUsage_reserved_Quota.String() == "0"
+
+}
+
+func validate_increasedQuota(ctx context.Context, diskSize int64, totalUsedQuotaBefore *resource.Quantity,
+	totalUsedQuotaAfterexpansion *resource.Quantity) bool {
+	var validTotalQuota bool
+	validTotalQuota = false
+
+	out := make([]byte, 0, 64)
+	result, suffix := totalUsedQuotaBefore.CanonicalizeBytes(out)
+	value := string(result)
+	quotaBefore, err := strconv.ParseInt(string(value), 10, 64)
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+	ginkgo.By(fmt.Sprintf(" quotaBefore :  %v%s", quotaBefore, string(suffix)))
+
+	result1, suffix1 := totalUsedQuotaAfterexpansion.CanonicalizeBytes(out)
+	value1 := string(result1)
+	quotaAfter, err := strconv.ParseInt(string(value1), 10, 64)
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	ginkgo.By(fmt.Sprintf(" quotaAfter :  %v%s", quotaAfter, string(suffix1)))
+
+	if string(suffix) == "Gi" {
+		var bytes int64 = diskSize
+		// Convert to Gi
+		//kibibytes := float64(bytes) / 1024
+		diskSize = int64(bytes) / 1024
+		fmt.Printf("diskSize:  %dGi\n", diskSize)
+
+	}
+
+	if quotaBefore < quotaAfter {
+		validTotalQuota = true
+		ginkgo.By(fmt.Sprintf("quotaBefore +diskSize:  %v, quotaAfter : %v", quotaBefore, quotaAfter))
+		ginkgo.By(fmt.Sprintf("validTotalQuota on storagePolicy:  %v", validTotalQuota))
+
+	}
+	return validTotalQuota
+}
+
+// stopCSIPods function stops all the running csi pods
+func stopKubeSystemPods(ctx context.Context, client clientset.Interface, namespace string) (bool, error) {
```

Comment:
Please rename this method to appropriate name and add a comment accordingly

Reply:
done